cypress/e2e/4-node-creator.cy.ts

```diff
@@ -315,6 +315,21 @@ describe('Node Creator', () => {
 		});
 	});
 
+	it.only('should correctly append a No Op node when Loop Over Items node is added', () => {
```

Comment:
should add more tests for case when there's already a trigger node.. found a bug there..
<img width="660" alt="Screenshot 2023-10-04 at 16 39 35" src="https://github.com/n8n-io/n8n/assets/4711238/c35ddae8-4b44-4353-aca0-3e3275de10bb">

Reply:
Good find! fixed and added e2e